content/docs/start/data-management/data-pipelines.md

```diff
@@ -57,10 +62,9 @@ Now let's install the requirements:
 
 ```dvc
 $ pip install -r src/requirements.txt
+$ git commit
```

Comment:
why do we need `git commit` here? It's unusual flow

overall, let's not overcomplicate pls. First priority is still simplicity and ability to get the "idea". It'll always contradict with making it more and more reproducible and correct.